jax/_src/numpy/util.py

```diff
@@ -310,6 +310,11 @@ def promote_dtypes_complex(*args: ArrayLike) -> list[Array]:
   return [lax._convert_element_type(x, to_dtype_complex, weak_type)
           for x in args]
 
+def _promote_to_default_integral_dtype(x: Array) -> Array:
```

Comment:
Since we're referencing this outside this module, the API shouldn't start with an underscore.

What would you think about changing this to `promote_dtypes_integer`, and make it follow the same API convention of the other `promote_dtypes` functions in this module?